### PR TITLE
CASMSEC-388: Document Kyverno Tooling to Report Audit Violations

### PR DESCRIPTION
**Pull request by pradeepkumargl**
The documentation change will be used by HPE internally and customers. This helps to identify the policy violations. There is no functionality impact with this documentation change.

### CODE REVIEW

**Comment by Viswanath-Murthy** [comment]
Kyverno baselines policies were introduced in CSM 1.4 which only logs the violation .Since the best security practices are validated in audit mode so there will not be any functional impact and in addition to that to ease the understanding of violations, we are providing a command that reports the violations that can be used to help further debugging.

**Review by jeremy-duckworth — APPROVED** [review]

File: operations/kubernetes/Kyverno.md [hunk 1]
```diff
@@ -323,6 +323,75 @@ Also, it generates the report of policy violation in respective workloads. The f
 
     If there are any discrepancies, look at the detailed policy report to triage the issue.
 
+## What is new in HPE CSM v1.4
```
Comment (nrockershousen) [inline operations/kubernetes/Kyverno.md:326]:
```suggestion
## What is new in the HPE CSM 1.4 release
```

File: operations/kubernetes/Kyverno.md [hunk 2]
```diff
@@ -323,6 +323,75 @@ Also, it generates the report of policy violation in respective workloads. The f
...
+
+As part of HPE CSM v1.4, the upstream Baseline profile is made available for the customer.
```
Comment (nrockershousen) [inline operations/kubernetes/Kyverno.md:328]:
```suggestion
The upstream Baseline profile is now available for customers as part of the HPE CSM 1.4 release.
```

File: operations/kubernetes/Kyverno.md [hunk 3]
```diff
@@ -323,6 +323,75 @@ Also, it generates the report of policy violation in respective workloads. The f
...
+disallow-host-process            true         audit    true
+disallow-privileged-containers   true         audit    true
+disallow-proc-mount              true         audit    true
+disallow-selinux                 true         audit    true
+restrict-apparmor-profiles       true         audit    true
+restrict-seccomp                 true         audit    true
+restrict-sysctls                 true         audit    true
+```
+
+The violations for each of the baseline policy is logged in policy report, similar to other policies as mentioned in Validation section above.
```
Comment (nrockershousen) [inline operations/kubernetes/Kyverno.md:358]:
```suggestion
The violations for each of the Baseline policies is logged in a policy report, similar to the other policies mentioned in Validation section above.
```

File: operations/kubernetes/Kyverno.md [hunk 4]
```diff
@@ -323,6 +323,75 @@ Also, it generates the report of policy violation in respective workloads. The f
...
+In order to give more insights towards each violation use below mentioned command.
```
Comment (nrockershousen) [inline operations/kubernetes/Kyverno.md:359]:
```suggestion
To get more information on each violation, use the following command.
```

File: operations/kubernetes/Kyverno.md [hunk 5]
```diff
@@ -323,6 +323,75 @@ Also, it generates the report of policy violation in respective workloads. The f
 
     If there are any discrepancies, look at the detailed policy report to triage the issue.
 
+## What is new in HPE CSM v1.4
+
+As part of HPE CSM v1.4, the upstream Baseline profile is made available for the customer.
+
+The Baseline profile is a collection of policies which implement the various levels of Kubernetes [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/).
+
+The Baseline profile is minimally restrictive and denies the most common vulnerabilities. It also follows many of the common security best practices for Pods.
```
Comment (nrockershousen) [inline operations/kubernetes/Kyverno.md:332]:
```suggestion
The Baseline profile is minimally restrictive and denies the most common vulnerabilities. It also follows many of the common security best practices for Kubernetes pods.
```

**Review by rustydb — APPROVED** [review]

**Comment by rustydb** [comment]
Does this need a backport to 1.5?

**Pull request merged**